WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadListItemState.kt

```diff
@@ -1,8 +1,13 @@
-package org.wordpress.android.ui.jetpack.backup
+package org.wordpress.android.ui.jetpack.backup.download
 
 import androidx.annotation.DrawableRes
 import org.wordpress.android.R
 import org.wordpress.android.ui.jetpack.BackupAvailableItemsProvider.BackupAvailableItemType
+import org.wordpress.android.ui.jetpack.backup.download.BackupDownloadListItemState.ViewType.ACTION_BUTTON
+import org.wordpress.android.ui.jetpack.backup.download.BackupDownloadListItemState.ViewType.CHECKBOX
+import org.wordpress.android.ui.jetpack.backup.download.BackupDownloadListItemState.ViewType.DESCRIPTION
+import org.wordpress.android.ui.jetpack.backup.download.BackupDownloadListItemState.ViewType.HEADER
+import org.wordpress.android.ui.jetpack.backup.download.BackupDownloadListItemState.ViewType.SUBHEADER
```

Comment:
Minor (🔍 ): Probably this was automatically done due to the way your `Optimize imports` works, not sure, we can check it out if you like. In the meanwhile, can you revert this change if that wasn't intentional?

Reply:
This is changed in a future PR. so all is good.